### PR TITLE
Change path to conclave repo

### PR DESCRIPTION
**Pull request by snedamle**

### CODE REVIEW

**Comment by gendal** [comment]
Hi Sneha - nice to see the additional polish on the samples!

Re the repo location, did the global gradle settings 'trick' not work?

File: README.md [hunk 1]
```diff
@@ -10,6 +10,46 @@ developers to get started with the conclave platform and understand different fe
...
+
+### Initial SetUp
+
+In order to compile, and run these samples successfully, kindly download the conclave SDK in conclave-samples/conclave-sdk/{version-number}/ folder.
+This sdk consists of a repo folder. You can find a reference to this repo path in gradle.properties as shown below.
+conclaveRepo=../conclave-sdk/1.1/repo. 
+
+If you wish to download the conclave sdk repo to another location, please update this conclaveRepo property in 
+gradle.properties for all the samples.
+
```
Comment (snedamle) [inline README.md:49]: @gendal I have mentioned in the Readme to download the SDK and save it in the conclave-samples/conclave-sdk/{version-number}/ 
and accordingly have updated the location in gradle to point to ../conclave-sdk/1.1/repo. 

So now all the samples will work if they have the sdk at this location conclave-samples/conclave-sdk/{version-number}/ 
This looked to be a quick fix. 

But let me know your thoughts on this.

Comment (gendal) [reply README.md:49]: Hi Sneha - I was referring to the discussion in the comments of this Jira: https://r3-cev.atlassian.net/browse/CON-574. @rusheb  outlined a global way to set the repo directory that _eliminated_ the need for users of the samples to have to edit multiple files.

Comment (snedamle) [reply README.md:49]: @gendal ok got you, will try this out.

**Pull request closed without merging**